README.rst

```diff
@@ -4,24 +4,15 @@ edx-when
 |pypi-badge| |travis-badge| |codecov-badge| |doc-badge| |pyversions-badge|
 |license-badge|
 
-The ``README.rst`` file should start with a brief description of the repository,
-which sets it in the context of other repositories under the ``edx``
-organization. It should make clear where this fits in to the overall edX
-codebase.
-
-Your project description goes here
-
-Overview (please modify)
-------------------------
-
-The ``README.rst`` file should then provide an overview of the code in this
-repository, including the main components and useful entry points for starting
-to understand the code in more detail.
-
-Documentation
--------------
-
-The full documentation is at https://edx-when.readthedocs.org.
```

Comment:
This is not a real link so definitely shouldn't be in here.

Reply:
We should get edx-when publishing to RTD, though, so that that can actually be a real link.

Reply:
Agreed. We will add it back in once we have that set up.